spec/controllers/controller_patches_spec.rb

```diff
@@ -9,7 +9,7 @@
   describe "caching blog content" do
 
     before do
-      allow(controller).to receive(:blog).and_return("")
+      allow(controller).to receive(:get_blog_content)
```

Comment:
`blog` is a controller action that returns a String but (at least under 0.25 and 0.26) `get_blog_content` assigns data to template variables rather than returning a value - the stub appears to be in place to avoid actually calling `quietly_try_to_open` which attempts to load XML from a remote site (previously called from within `blog` but it looks as though I failed to update the spec to match the [change in behaviour](https://github.com/mysociety/asktheeu-theme/commit/a545acc2671ca19854c3106f461dd032c2bafe5e))